package.json

```diff
@@ -31,22 +31,22 @@
     "@angular/common": "~8.2.1",
     "@angular/compiler": "~8.2.1",
     "@angular/core": "~8.2.1",
-    "@angular/forms": "~8.2.1",
+    "@angular/forms": "^8.2.14",
```

Comment:
why this specific /forms is having different version?

Reply:
"@angular/forms": "^8.2.14", is required as per the latest commits